src/translations/en.json

```diff
@@ -699,13 +699,17 @@
             "add_automation": "Add automation"
           },
           "editor": {
-            "introduction": "Use automations to bring your home alive",
+            "introduction": "Use automations to bring your home alive. Automations rules can be complicated, optionally add a description to document what it does and how.",
```

Comment:
Let's drop this. It's the least important feature on this page and so should not require an individual call out.

Reply:
OK, removed!